Blish HUD/Controls/TextInputBase.cs

```diff
@@ -568,10 +560,10 @@ public abstract class TextInputBase : Control {
 
                 ClipboardUtil.WindowsClipboardService.SetTextAsync(clipboardText)
                              .ContinueWith((clipboardResult) => {
-                                  if (clipboardResult.IsFaulted) {
-                                      Logger.Warn(clipboardResult.Exception, "Failed to set clipboard text to {clipboardText}!", clipboardText);
-                                  }
-                              });
+                                 if (clipboardResult.IsFaulted) {
```

Comment:
It might be better to use
```csharp
(!clipboardResult.IsCompleted)
```
instead in a ContinueWith since IsCompleted covers 
Canceled, Faulted and RanToCompletion according to [docs](https://docs.microsoft.com/de-de/dotnet/api/system.threading.tasks.task.iscompleted?redirectedfrom=MSDN&view=net-5.0#System_Threading_Tasks_Task_IsCompleted).

Reply:
This wasn't part of the original PR - it was just a formatting thing that got included.  We also only want to log an entry if it strictly faulted.  If it gets canceled or otherwise, no reason to log it because there won't be an exception included.